db/seeders/distributions_seeder.rb

```diff
@@ -10,15 +10,12 @@ def initialize(organization)
   end
 
   def seed
-    20.times.each do
-      storage_location = random_record_for_org(organization, StorageLocation)
-      stored_inventory_items_sample = storage_location.inventory_items.sample(20)
+    storage_location = random_record_for_org(organization, StorageLocation)
 
+    storage_location.inventory_items.sample(20).each do |stored_inventory_item|
       distribution = create_distribution(storage_location)
+      create_line_item(stored_inventory_item, distribution)
 
-      stored_inventory_items_sample.each do |stored_inventory_item|
-        create_line_item(stored_inventory_item, distribution)
-      end
       distribution.reload
       distribution.storage_location.decrease_inventory(distribution)
     end
```

Comment:
The outer loop of `20.times` has been removed in this commit. Is this intentional? I think this will result in 20x fewer records.